CHANGELOG.md

```diff
@@ -1,5 +1,44 @@
 # TigerBeetle Changelog
 
+## 2023-10-25
+
+### Features
+
+- [#1245](https://github.com/tigerbeetle/tigerbeetle/pull/1245)
+
+  Include docker images into the release.
+
+### Correctness And Performance
+
+- [#1240](https://github.com/tigerbeetle/tigerbeetle/pull/1240)
+
+  Increase block size to [reduce latencies](https://open.spotify.com/track/30oZqbcUROFLSru3WcN3bx)
```

Comment:
People might follow the "information scent" of "reduce latencies" and expect to find benchmark results. Perhaps move the Easter egg to the last paragraph:

> Speed up secondary index updates by statically distinguishing between insertions and
   updates. [Faster than the speed of night!](https://open.spotify.com/track/30oZqbcUROFLSru3WcN3bx)
```suggestion
  Increase block size to [reduce latencies](https://open.spotify.com/track/30oZqbcUROFLSru3WcN3bx)
```